pyanaconda/ihelp.py

```diff
@@ -96,6 +96,15 @@ def get_help_path(help_file, instclass, plain_text=False):
 
     :return str: full path to the help file requested or to a placeholder
     """
+
+    if instclass.name.startswith("Red Hat "):
+        # NOTE: Help content is not yet available for RHEL, so just always return
+        #       a placeholder for now.
+        if plain_text:
+            return "/usr/share/anaconda/rhel_help_tui_placeholder.txt"
+        else:
+            return "/usr/share/anaconda/rhel_help_gui_placeholder.xml"
```

Comment:
I think, that if you change `help_folder`, `help_placeholder` and `help_placeholder_plain_text` in the install class for RHEL, the `get_help_path` function might work without these modification.